### PR TITLE
add dispora item to sharer in sharer.js

### PR DESCRIPTION
**Pull request by mohmmadAyesh**
Hi I added dispora item to sharer link and test it locally using sharer.js embeded in index.html and it works hope you accept it and merge this PR to continue add section for it in index.html also a link to it in top of page after it been added to sharer.min.js
Thanks.

### CODE REVIEW

**Comment by mohmmadAyesh** [comment]
@ellisonleao  can you review this PR and merge it Please

File: sharer.js [hunk 1]
```diff
@@ -42,6 +42,7 @@
     var target = elem.currentTarget || elem.srcElement;
     var sharer = new Sharer(target);
     sharer.share();
+
```
Comment (ellisonleao) [inline sharer.js:45]: please remove this extra space

File: sharer.js [hunk 2]
```diff
@@ -455,7 +459,7 @@
           window.location.href = sharer.shareUrl;
         }
       } else {
-        console.log(sharer.shareUrl);
+        console.log(sharer.params);
```
Comment (ellisonleao) [inline sharer.js:462]: can you roll back this change, please?

**Review by ellisonleao — CHANGES_REQUESTED** [review]
hey @mohmmadAyesh thanks for the PR. Just a few minor changes and we should be fine.

**Comment by mohmmadAyesh** [comment]
@ellisonleao here I did as you require can you please merge this PR if you dont have any further changes and I would be thankful

**Review by ellisonleao — APPROVED** [review]

**Pull request merged**